routes/web.php

```diff
@@ -47,6 +47,11 @@
     Route::get('login/{provider}', 'redirectToProvider')->name('login.redirect');
     Route::get('login/{provider}/callback', 'handleProviderCallback');
     Route::get('logout', 'logout')->name('logout');
+    Route::post('login/gsm', 'loginWithPhoneNumber')->name('login.gsm');
+    Route::get('verification', 'verificationPage')->name('verification.page');
```

Comment:
could you please change the route to `GET login/gsm/verify` & `POST login/gsm/verify` for another one to be aligned with RESTful naming